Update framework for 2017

vincenzocoia opened the pull request:
By "framework" I mean things like:

- Updated dates throughout website.
- Removed links to homework.
- Removed reference to guest lecturers

I didn't change the actual course content. The way I see it, the course is "ready"-ish the way it is, aside from class-specific things like the "Who's in here?" table in cm003.

jennybc commented:
I think I already had a fairly decent separation like this:

  * Course content, not highly perishable. I think basically what we see here: <http://stat545.com/topics.html>
  * Stuff specific to a certain run of the course, i.e. landing pages for specific class meetings. Basically what we see here: <http://stat545.com/syllabus.html>

Do you concur?
And can you make a clean statement about, for example, that the PR affects the syllabus page and things below? And does not affect the topics pages and things below?

It touches 118 files, so this is easier to work out via human conversation 🙂

vincenzocoia commented:
Yep, that looks like a good division. Indeed this pull request only changes files on the syllabus (your second bullet point).

The files changed:

- __53 files__: I copied the 2016 `html` and `md` versions of all the `cmXXX` files over to a new `cm_2016` folder
- __52 files__: I modified dates and no-longer-accurate statements from the main `cmXXX` `html` and `md` files
- __8 files__: Modified dates on homework
- __3 files__: Same sort of changes to the `faq`, `people`, `syllabus`
- __2 files__: `.js` files as a result of `rmarkdown::render`ing.

Sum = 118

Upon checking these, I realized I forgot to render the faq, people, and syllabus. Silly me. I'll close this and issue another pull request with those rendered. Sorry about that.
